artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/ActionAbstract.java

```diff
@@ -113,7 +115,23 @@ public String getBrokerURLInstance(String acceptor) {
                   return new URI(scheme, null, host, port, null, null, null).toString();
                }
             }
+
+            for (TransportConfiguration connectorConfiguration: brokerConfiguration.getConnectorConfigurations().values()) {
+               if (connectorConfiguration.getName().equals(acceptor)) {
+                  Map<String, Object> acceptorParams = connectorConfiguration.getParams();
+                  String scheme = ConfigurationHelper.getStringProperty(TransportConstants.SCHEME_PROP_NAME, SchemaConstants.TCP, acceptorParams);
+                  String host = ConfigurationHelper.getStringProperty(TransportConstants.HOST_PROP_NAME, "localhost", acceptorParams);
+                  int port = ConfigurationHelper.getIntProperty(TransportConstants.PORT_PROP_NAME, 61616, acceptorParams);
+
+                  if (InetAddress.getByName(host).isAnyLocalAddress()) {
+                     host = "localhost";
+                  }
+
+                  return new URI(scheme, null, host, port, null, null, null).toString();
+               }
+            }
          } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Why is not the stack trace printed using `getActionContext().out` ?

Reply:
I didn't mean to commit this line.  Will remove it.

Reply:
I will actually keep the e.printStackTrace with your suggestion.